### PR TITLE
Update to sbt-lucuma 0.6

### PR DESCRIPTION
**Pull request by armanbilge**

### CODE REVIEW

File: modules/jts-awt/src/main/java/org/locationtech/jts/awt/FontGlyphReader.java [hunk 1]
```diff
@@ -1,13 +1,6 @@
...
- *
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * and Eclipse Distribution License v. 1.0 which accompanies this distribution.
- * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
- * and the Eclipse Distribution License is available at
- *
- * http://www.eclipse.org/org/documents/edl-v10.php.
+ * Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+ * For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
```
Comment (armanbilge) [inline modules/jts-awt/src/main/java/org/locationtech/jts/awt/FontGlyphReader.java:3]: Uh oh.

**Comment by armanbilge** [comment]
Phew 😓

File: modules/jts-awt/src/main/java/org/locationtech/jts/awt/FontGlyphReader.java [hunk 2]
```diff
@@ -1,3 +1,8 @@
+/*
```
Comment (cquiroz) [inline modules/jts-awt/src/main/java/org/locationtech/jts/awt/FontGlyphReader.java:1]: Seems odd to add our copyright for code that was imported

Comment (armanbilge) [reply modules/jts-awt/src/main/java/org/locationtech/jts/awt/FontGlyphReader.java:1]: Oh yes, I thought I removed those. Let me try again.

Comment (armanbilge) [reply modules/jts-awt/src/main/java/org/locationtech/jts/awt/FontGlyphReader.java:1]: Is this whole module imported?

Comment (cquiroz) [reply modules/jts-awt/src/main/java/org/locationtech/jts/awt/FontGlyphReader.java:1]: All java code was copied from the original jts

**Review by cquiroz — APPROVED** [review]
LGTM thanks
